crates/red_knot/src/module.rs

```diff
@@ -566,7 +575,10 @@ mod tests {
         let foo_module = resolver.module(foo_id.unwrap());
 
         assert_eq!(&src, foo_module.path().root());
-        assert_eq!(&foo_stub, &*files.path(foo_module.path().file()));
+        assert_eq!(
+            &foo_stub.canonicalize()?,
+            &*files.path(foo_module.path().file())
+        );
```

Comment:
Hmm this is kind of annoying. It now tests more than it should. Anyway, I think we should add a helper for the assertion to avoid that future us add a test that incorrectly compares paths in the future

Reply:
Actually. I think the better idea is to fixup the `TestCase` construction and canonicalize the directory paths there (once)